ui/src/api/log.js

```diff
@@ -0,0 +1,14 @@
+import api from '.';
+
+const endpoint = api.url('/log');
+
+export function fetchLogMessages() {
+  return endpoint.get('/').json();
+}
+
+export function sendLogFrontEndTraceBack(stack, state) {
+  const { logger, ...stateToLog } = state;
```

Comment:
I use destructuring to exclude `logger` from the `stateToLog` object sent to the server (instead of `delete` previously).

Reply:
:+1: